Formula/proji.rb

```diff
@@ -0,0 +1,17 @@
+class Proji < Formula
+  desc "Powerful cross-platform CLI project templating tool"
+  homepage "https://github.com/nikoksr/proji"
+  url "https://github.com/nikoksr/proji/releases/download/v0.19.0/proji-macOS-64bit.tar.gz"
+  version "0.19.0"
+  sha256 "a9a488650fd4a964c4cccf4d816e04cb6886f94f50fa1d0fefa14be033f01c5b"
+  bottle :unneeded
+
+  def install
+    bin.install "proji"
+    system "#{bin}/proji", "init"
+  end
+
+  test do
+    system "#{bin}/proji", "version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
Same if the software requires a virtual machine, docker instance, etc. to be running.